white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/model/Account.java

```diff
@@ -25,7 +25,7 @@
     private String passwordHash;
 
     @Enumerated(STRING)
-    private Role role;
+    private Role2 role;
```

Comment:
jak dla mnie możesz dać tutaj tą swoją klase `Role` , a `Role2` wywalic. Te klasy które stworzyłem to tylko był pomysł, ale nie musimy sie go tak szytwno trzymać